Documentation/PackedAssetBuilds.md

```diff
@@ -0,0 +1,122 @@
+
+> REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
+
+### Build profile build and process assets
+
+Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+
+### Test the profile build
+
+* Open the game in editor
+    * load `NewStarBase` level
+    * Verify that game can launch and connect to local server from editor
+* Validate local game launcher can connect to local server
+
+### Build AssetBuilder
+
+You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built.
+
+For example:
+```shell
+cmake --build build\windows --target AssetBundler --config profile -- /m /nologo
+```
+
+### Build monolithic game
+
+Build a second version of the executables as monolithic pak builds.
+
+``` shell
+# Create build files for a monolithic build that also disables all user/project registry settings overrides
+cmake -B build\windows_mono -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=c:\your\path\to\3rdParty -DLY_MONOLITHIC_GAME=1 -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0
+
+# Build the profile versions of all the executables
+cmake --build build\windows_mono --target MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher MultiplayerSample.UnifiedLauncher --config profile -- /m /nologo
+```
+
+The outputs in windows_mono\bin\profile can be copied and run anywhere, once the pak files are put in the proper location.
+
+
+### Bundle Content
+
+Run the AssetBundler
+
+```
+build\windows\bin\profile\AssetBundler.exe --project-path="c:\your\path\to\o3de-multiplayersample"
+```
+
+Follow steps for "Create a bundle for game assets", "Create a bundle for engine assets" and "Add bundles to the release game layout" from https://www.o3de.org/docs/user-guide/packaging/asset-bundler/bundle-assets-for-release/
+
+* The "default seed lists" choice should choose all but 4 seed lists to make the `engine_pc.pak`
+* The other [seed lists](https://github.com/o3de/o3de-multiplayersample/tree/development/AssetBundling/SeedLists) should all get selected to make the `game_pc.pak`.
+
+> It's important to make sure that the bootstrap.game.profile.setreg file has been added to one of the seed files. (also add debug if you want to support debug builds)
+
+### Create the Launcher Zip file
+
+Use the following .bat file or equivalent copy steps to create a directory with the launchers in it:
+```shell
+rem Use this by calling 'make_release D:\my\output\dir' to make a release directory
+mkdir %1
+mkdir %1\Cache
+mkdir %1\Cache\pc
+mkdir %1\Gems
+mkdir %1\Gems\AWSCore
+ 
+rem Copy the pak files
+copy c:\your\path\to\o3de-multiplayersample\AssetBundling\Bundles\*.pak %1\Cache\pc
+ 
+rem Copy the executables and DLLs
```

Comment:
The *.* will also copy in gigs of debug symbol pdb’s might call out removing them by hand or update the script

Reply:
Good call out, for now will add a note